src/components/icon/WIcon.vue

```diff
@@ -1,32 +1,34 @@
 <template>
-  <component :class="{ color: color }" :is="iconComponent" v-bind="$attrs" />
+  <component :class="`${fillColor}`" :is="iconComponent" v-bind="$attrs" />
```

Comment:
Right arrow is not visible because it is black
![image](https://github.com/user-attachments/assets/a9ec4747-11a3-410d-a46e-79042368c7c7)